### ISSUE TITLE
FR: Support for BT Thermocouples/Thermometers

### ISSUE THREAD

**Issue by mylesagray** [graphefruit/Beanconqueror#517]
SEP has support for some Bluetooth Thermo-devices for logging temperature alongside pressure and weight, it would be really neat if BQ could do the same.

The thermocouples they currently support are the:

 * Thermapen ThermaQ Blue Thermometer with Bluetooth LE
 * BlueTherm One LE thermometer

Interestingly, from the photos they look like they're made by exactly the same OEM - so perhaps there are cheaper non-branded options out there.

**Comment by graphefruit** [comment]
Hey @mylesagray,
actually I think every bluetooth term device is possible if the API is open and documented.

But without having the devices at my side, I don't think I can do any integration here.
If you're going to integrate them, I could give you a helping hand atleast.

**Comment by mylesagray** [comment]
I’ve requested the SDK from ETI (who make the devices) as they state it’s available on request.

Awaiting the device and will do some digging once it arrives and see what I can do to integrate it.

**Comment by graphefruit** [comment]
Great to hear!
I could offer a pair-programming session, and when the device/api is working.
I could go and write the initial code if this would help

**Comment by mylesagray** [comment]
That would be very helpful :)We can organise on Discord once the hardware arrives and I get familiar with the code base.  On Tue, Apr 11, 2023 at 19:56, Lars Saalbach ***@***.***> wrote:  
Great to hear!
I could offer a pair-programming session, and when the device/api is working.
I could go and write the initial code if this would help

**Comment by graphefruit** [comment]
@mylesagray I'd say this is done for the actual integration part?

**Comment by mylesagray** [comment]
Yup I’d say this is good to close.  On Mon, May 8, 2023 at 20:34, Lars Saalbach ***@***.***> wrote:  
@mylesagray I'd say this is done for the actual integration part?
